_overviews/scala3-book/scala-for-python-devs.md

```diff
@@ -1291,6 +1291,29 @@ Follow the links below for more details:
 - Infix methods
 - Macros and metaprogramming
 
+## How to setup a virtual environnement in Scala?
+
```

Comment:
I'd love if there were a "most important takeaway" right on top before you get to the introduction about what python does.

In scala, there is no need for virtual environments.

Then after that comes a description of the python situation and how building with sbt differs from that. A reader could stop reading after that first sentence, or read more if they want to convince themselves it's true.